src/helpers/uniswap-v3/move-to-shared/token-amounts.ts

```diff
@@ -0,0 +1,47 @@
+import { type Pool, Position } from '@uniswap/v3-sdk'
+import { BigNumber } from '@ryze-blockchain/ethereum'
+import { parseUnits } from 'ethers'
+
+export function getAmounts(
```

Comment:
i think this should be called something like `getLiquidityAmounts`, also since we have many univ3 helpers, it would make sense to add them to a class so we dont have to call it `getUniswapV3LiquidityAmounts` and instead call `UniswapV3.getLiquidityAmounts`